xds/internal/xdsclient/bootstrap/bootstrap.go

```diff
@@ -43,9 +43,13 @@ import (
 
 const (
 	// The "server_features" field in the bootstrap file contains a list of
-	// features supported by the server. A value of "xds_v3" indicates that the
-	// server supports the v3 version of the xDS transport protocol.
-	serverFeaturesV3 = "xds_v3"
+	// features supported by the server.
+	// + A value of "xds_v3" indicates that the server supports the v3 version of
```

Comment:
What does this `+` do? Anything fancy?

Reply:
I think in most places we use as a dash or an asterisk. This is not part of an exported API, but godoc does specify some syntax for comments. Dashes are used for lists. See: https://tip.golang.org/doc/comment.